website/docs/sequence.md

```diff
@@ -0,0 +1,430 @@
+
+Now that we've populated our `x` axis, let's move on with the `y` axis.
+
+The main way to move forward on the `y` axis in a sequence diagram is through `events`.\
+An `event` needs a name to refer to it later on and automatically moves downward on the y axis:
+
+```hylimo
+sequenceDiagram {
+    event("startPayment")
+}
+```
+
+As you can see above, the name should be understandable for you to know exactly which action is symbolized by this event.
+This name will never be shown anywhere.
+
+For most cases, the default value for how far away the next event will be good-enough.
+Nevertheless, in cases where you want an explicit distance to the predecessor, you are free to do so:
+
+```hylimo
+sequenceDiagram {
+    event("startPayment", 50)
+}
+```
+
+This positions the event `50` pixels below its predecessor.
+Note that while negative values are allowed here, they won't make any sense:
+Instead, you can also move the previous event up by that amount and invert their meanings.
+
+Both `event` and `participant` names will be registered as variables if they didn't exist already.
+In case they existed already, the existing name takes precedence.
+If their name is already used by something else, you can assign the result of these functions to a variable of your own choosing:
+
+```hylimo
+sequenceDiagram {
+    enableDebugging = true
+    Bob = true
+    user = instance("Bob")
+    event("buy")
+    event("stop")
+    buy.Bob --> stop.Bob
+}
+```
+
+In this example, we also see that to refer to a specific xy-coordinate, we can always use `eventname.participant`.
+Additionally, we can enable the debugging mode to understand what's happening inside our diagram by setting `enableDebugging = true`.
+
+Now, we have the basic knowledge to go on with the remaining features that are all relative to the latest event.
+
+## Interacting with participants
+
+There is a bunch of things you can do with participants, depending on if there is a latest defined event:
+
+- you can postpone the creation of a participant by defining it after an event:
+
+```hylimo
+sequenceDiagram {
+    instance("A")
+    event("smth")
+    instance("B")
+}
+```
+
+- You can destroy a participant:
+
+```hylimo
+sequenceDiagram {
+    instance("A")
+    instance("B")
+    event("E1")
+    destroy(A)
+    event("E2")
+}
+```
+
+- You can reanimate a dead participant:
+
+```hylimo
+sequenceDiagram {
+    instance("A")
+    instance("B")
+    event("E1")
+    destroy(A)
+    event("E2")
+    event("E3")
+    instance("A²", below = A)
+}
+```
+
+- You can activate and deactivate a participant, meaning that it is actively working on something:
+
+```hylimo
+sequenceDiagram {
+    instance("A")
+    instance("B")
+    event("E1")
+    activate(A)
+    event("E2")
+    deactivate(A)
+}
+```
+
+- Even multiple times simultaneously:
+
+```hylimo
+sequenceDiagram {
+    instance("A")
+    instance("B")
+    event("E1")
+    activate(A)
+    event("E1margin", 5)
+    activate(A)
```

Comment:
I get an error here

Reply:
Fixed. Came from the name change (`xshift -> xShift`)